src/App.js

```diff
@@ -1,25 +1,41 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+/* eslint-disable comma-dangle */

```

Comment:
- [ ] Kindly, remove the old commented code found on these files:

```
login.jsx
navbar.module.scss
MainPage.js
MainPage.module.scss
useAxios.js
NavContainer.module.scss
RootLayout.module.scss
```

That way, the repo's cleanliness will be improved. 👍

Reply:
@Meltrust Thank you for your review. I have tried to find out these commented codes in App.js and all other files, but could not find them. I guess you might have seen an old commit that contained these codes.